quickwit-metastore/docker-compose.yml

```diff
@@ -1,9 +1,10 @@
-version: "3.9" 
+version: "3.9"
```

Comment:
I recently committed a docker-compose file at the root of the project; we can work off that one and delete this one.